plugin/http/instrumentation/client.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package instrumentation
+
+import (
+	"net/http"
+
+	"go.opencensus.io/trace/propagation"
+)
+
+// Transport is an http.RoundTripper that instruments all outgoing requests with
+// stats and tracing. The zero value is intended to be a useful default, but for
+// now it's recommended that you explicitly set Propagation.
+type Transport struct {
+	// Base may be set to wrap another http.RoundTripper, by default
+	// http.DefaultTransport is used.
+	Base http.RoundTripper
+	// NoStats may be set to disable recording of stats.
+	NoStats bool
+	// NoTrace may be set to disable both recording of traces.
```

Comment:
What did you mean by 'disable both recording of traces.'? Perhaps this is an incomplete comment?